src/algorithms/signal_source/gnuradio_blocks/fifo_reader.cc

```diff
@@ -61,12 +67,14 @@ int FifoReader::work(int noutput_items,
     size_t items_retrieved = 0;
     if (sample_type_ == "ishort")
         {
-            // ishort == int16_t
```

Comment:
that's not a bad comment. Why remove it?

Reply:
Put back in [77c682e](https://github.com/gnss-sdr/gnss-sdr/pull/584/commits/77c682e8b312c645d77e60289b0d67b33e67c633).